scripts/us_census/acs5yr/subject_tables/common/testdata/colMapInputs/spec.json

```diff
@@ -0,0 +1,240 @@
+			"Information": "NAICS/51",
+			"Finance and insurance, and real estate and rental and leasing": "NAICS/52-53",
+			"Professional, scientific, and management, and administrative and waste management services": "NAICS/54-56",
+			"Educational services, and health care and social assistance": "NAICS/61-62",
+			"Arts, entertainment, and recreation, and accommodation and food services": "NAICS/71-72",
+			"Other services (except public administration)": "NAICS/81",
+			"Public administration": "NAICS/82"
+		},
+		"occupation": {
+			"Sales and office occupations": "SalesAndOfficeOccupations",
+			"Service occupations": "ServiceOccupations",
+			"Management, business, science, and arts occupations": "ManagementOrBusinessOrScienceOrArtsOccupations",
+			"Natural resources, construction, and maintenance occupations": "NaturalResourcesOrConstructionOrMaintenanceOccupations",
+			"Production, transportation, and material moving occupations": "ProductionOrTransportationOrMaterialMovingOccupations"
+		},
+		"earnings": {
+			"$1 to $4,999 or loss": "[- 4999 USDollar]",
+			"$5,000 to $14,999": "[5000 14999 USDollar]",
+			"$15,000 to $24,999": "[15000 24999 USDollar]",
+			"$25,000 to $34,999": "[25000 34999 USDollar]",
+			"$35,000 to $49,999": "[35000 49999 USDollar]",
+			"$50,000 to $74,999": "[50000 74999 USDollar]",
+			"$75,000 or more": "[75000 - USDollar]"
+		},
+		"income": {
+			"Under $25,000": "[- 25000 USDollar]",
+			"$25,000 to $49,999": "[25000 49999 USDollar]",
+			"$50,000 to $74,999": "[50000 74999 USDollar]",
+			"$75,000 to $99,999": "[75000 99999 USDollar]",
+			"$100,000 and over": "[100000 - USDollar]"
+		},
+		"ratioToPovertyLine": {
+			"Below 50 percent of the poverty level": "[- 0.5 RatioToPovertyLine]",
+			"50 to 99 percent of the poverty level": "[0.5 0.99 RatioToPovertyLine]",
+			"100 to 149 percent of the poverty level": "[1 1.49 RatioToPovertyLine]",
+			"150 to 199 percent of the poverty level": "[1.5 1.99 RatioToPovertyLine]",
+			"200 to 299 percent of the poverty level": "[2.0 2.99 RatioToPovertyLine]",
+			"At or above 300 percent of the poverty level": "[3 - RatioToPovertyLine]"
+		},
+		"healthInsurace": {
+			"Uninsured": "NoHealthInsurance"
+		}
+	},
+	"enumSpecializations": {
+		"Under 6 years": "Under 19 years",
+		"6 to 18 years": "Under 19 years",
+		"19 to 25 years": "19 to 64 years",
+		"26 to 34 years": "19 to 64 years",
+		"35 to 44 years": "19 to 64 years",
+		"45 to 54 years": "19 to 64 years",
+		"55 to 64 years": "19 to 64 years",
+		"65 to 74 years": "65 years and older",
+		"75 years and older": "65 years and older",
+		"White alone": "One race",
+		"Black or African American alone": "One race",
+		"American Indian and Alaska Native alone": "One race",
+		"Asian alone": "One race",
+		"Native Hawaiian and Other Pacific Islander alone": "One race",
+		"Some other race alone": "One race",
+		"Different House in the U.S.": "DifferentHouse1YearAgo",
```

Comment:
Are we mixing enum and column name?

Reply:
Good catch! I thought I fixed all the places but seems like I missed this one. I have fixed this.